src/components/ArtifactsPreview/ArtifactsPreview.js

```diff
@@ -70,6 +71,10 @@ const ArtifactsPreview = ({ artifact }) => {
       <h1>Sorry, something went wrong.</h1>
       <h3>We're working on it and we'll get it fixed as soon as we can.</h3>
     </div>
+  ) : !preview.length ? (
```

Comment:
Please be more explicit about the `length` property being of type `number` and use `=== 0` instead of `!`:
```suggestion
  ) : preview.length === 0 ? (
```

Reply:
done